// @bit/heyitsgeo.ng-semantic-ui.style-links/dist/bundles/bit-style-links.umd.js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core'), require('@angular/common')) :
    typeof define === 'function' && define.amd ? define('@bit/heyitsgeo.ng-semantic-ui.style-links', ['exports', '@angular/core', '@angular/common'], factory) :
    (global = global || self, factory((global.bit = global.bit || {}, global.bit['style-links'] = {}), global.ng.core, global.ng.common));
}(this, (function (exports, core, common) { 'use strict';

    /**
     * @fileoverview added by tsickle
     * Generated from: style-links.component.ts
     * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
     */
    var StyleLinksComponent = /** @class */ (function () {
        function StyleLinksComponent() {
        }
        StyleLinksComponent.decorators = [
            { type: core.Component, args: [{
                        selector: 'ng-semantic-style-links',
                        template: "<link rel=\"stylesheet\" href=\"https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css\">\n",
                        styles: [""]
                    }] }
        ];
        return StyleLinksComponent;
    }());

    /**
     * @fileoverview added by tsickle
     * Generated from: style-links.module.ts
     * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
     */
    var StyleLinksModule = /** @class */ (function () {
        function StyleLinksModule() {
        }
        StyleLinksModule.decorators = [
            { type: core.NgModule, args: [{
                        declarations: [StyleLinksComponent],
                        imports: [
                            common.CommonModule
                        ],
                        exports: [
                            StyleLinksComponent
                        ]
                    },] }
        ];
        return StyleLinksModule;
    }());

    exports.StyleLinksModule = StyleLinksModule;
    exports.ɵa = StyleLinksComponent;

    Object.defineProperty(exports, '__esModule', { value: true });

})));
