// @bit/heyitsgeo.ng-semantic-ui.style-links/dist/fesm2015/bit-style-links.js
import { Component, NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';

/**
 * @fileoverview added by tsickle
 * Generated from: style-links.component.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
class StyleLinksComponent {
}
StyleLinksComponent.decorators = [
    { type: Component, args: [{
                selector: 'ng-semantic-style-links',
                template: "<link rel=\"stylesheet\" href=\"https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css\">\n",
                styles: [""]
            }] }
];

/**
 * @fileoverview added by tsickle
 * Generated from: style-links.module.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
class StyleLinksModule {
}
StyleLinksModule.decorators = [
    { type: NgModule, args: [{
                declarations: [StyleLinksComponent],
                imports: [
                    CommonModule
                ],
                exports: [
                    StyleLinksComponent
                ]
            },] }
];

/**
 * @fileoverview added by tsickle
 * Generated from: bit-style-links.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */

export { StyleLinksModule, StyleLinksComponent as ɵa };
