// @bit/heyitsgeo.ng-semantic-ui.style-links/__ivy_ngcc__/dist/fesm2015/bit-style-links.js
import { Component, NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';

/**
 * @fileoverview added by tsickle
 * Generated from: style-links.component.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import * as ɵngcc0 from '@angular/core';
class StyleLinksComponent {
}
StyleLinksComponent.ɵfac = function StyleLinksComponent_Factory(t) { return new (t || StyleLinksComponent)(); };
StyleLinksComponent.ɵcmp = ɵngcc0.ɵɵdefineComponent({ type: StyleLinksComponent, selectors: [["ng-semantic-style-links"]], decls: 1, vars: 0, consts: [["rel", "stylesheet", "href", "https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css"]], template: function StyleLinksComponent_Template(rf, ctx) { if (rf & 1) {
        ɵngcc0.ɵɵelement(0, "link", 0);
    } }, styles: [""] });
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(StyleLinksComponent, [{
        type: Component,
        args: [{
                selector: 'ng-semantic-style-links',
                template: "<link rel=\"stylesheet\" href=\"https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css\">\n",
                styles: [""]
            }]
    }], null, null); })();

/**
 * @fileoverview added by tsickle
 * Generated from: style-links.module.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
class StyleLinksModule {
}
StyleLinksModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: StyleLinksModule });
StyleLinksModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function StyleLinksModule_Factory(t) { return new (t || StyleLinksModule)(); }, imports: [[
            CommonModule
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(StyleLinksModule, { declarations: function () { return [StyleLinksComponent]; }, imports: function () { return [CommonModule]; }, exports: function () { return [StyleLinksComponent]; } }); })();
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(StyleLinksModule, [{
        type: NgModule,
        args: [{
                declarations: [StyleLinksComponent],
                imports: [
                    CommonModule
                ],
                exports: [
                    StyleLinksComponent
                ]
            }]
    }], null, null); })();

/**
 * @fileoverview added by tsickle
 * Generated from: bit-style-links.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */

export { StyleLinksModule, StyleLinksComponent as ɵa };
